plugins/SitesManager/API.php

```diff
@@ -1266,14 +1277,19 @@ public function setKeepURLFragmentsGlobal($enabled)
      * Will also apply to websites created in the future.
      *
      * @param string $excludedQueryParameters Comma separated list of URL query parameters to exclude from URLs
+     * @deprecated Use self::setGlobalQueryParamExclusion() instead.
      * @return bool
      */
     public function setGlobalExcludedQueryParameters($excludedQueryParameters)
     {
-        Piwik::checkUserHasSuperUserAccess();
-        $excludedQueryParameters = $this->checkAndReturnCommaSeparatedStringList($excludedQueryParameters);
-        Option::set(self::OPTION_EXCLUDED_QUERY_PARAMETERS_GLOBAL, $excludedQueryParameters);
-        Cache::deleteTrackerCache();
+        if (empty($excludedQueryParameters)) {
+            $this->setGlobalQueryParamExclusion(SitesManager::URL_PARAM_EXCLUSION_TYPE_NAME_NO_EXCLUSIONS);
+            return;
```

Comment:
```suggestion
            return true;
```

To match the documented return type.